content/src/Server.ts

```diff
@@ -49,7 +49,9 @@ export class Server {
 
     this.app = express()
 
-    this.metricsServer = initializeMetricsServer(this.app, metricsComponent)
+    if (env.getConfig(EnvironmentConfig.METRICS)) {
```

Comment:
What is the case in which you _don't_ want metrics that is good enough to add an if in the code?

Reply:
This comes from @guidota to fix the running of the service, maybe it's not needed anymore

Reply:
for testing purposes we don't want to initialize multiple metrics servers (it fails locally), but did some changes to avoid using this environment config and just skip it for test servers

see #694